### PR TITLE
CLOUD-265-PXC-Release-Notes-1.0.0 added RN for the 1.0.0 version of K…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ubernetes Operator for PXC

### CODE REVIEW

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. rn:: 1.0.0
+
+Percona Kubernetes Operator for XtraDB Cluster
+==============================================
+
+Percona announces the general availability of |Percona Kubernetes Operator for XtraDB Cluster| 1.0.0 on May 24, 2019. This release is now the current GA release in the 1.0 series. Download the latest version from the Percona Software Repositories. Please see the `GA release announcement`. All of Percona's software is open-source and free.
+
+The Percona Kubernetes Operator for Percona XtraDB Cluster automates the creation, modification, and deletion of members of your Percona XtraDB Cluster environment. The Operator can be used to instantiate a Percona XtraDB Cluster, or scale an existing Cluster.
+
+The Operator contains the necessary Kubernetes settings and provides a consistent Percona XtraDB Cluster instance. The Percona Kubernetes Operators are based on best practices for configuration and setup of the Percona XtraDB Cluster.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: The operator doesn't contain settings :(

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  * Automate backups or perform on-demand backups as needed with support for performing a restore
+  * Automate the recovery from failure of a single Percona XtraDB Cluster node
+  * Provide data encryption in the cluster and between the application and the nodes
+  * Access private registries to enhance security
+
+
+Installation
+------------
+
+Installation is performed by accessing the `Percona Software Repositories <https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html>`__ for Kubernetes and `OpenShift <https://www.percona.com/doc/kubernetes-operator-for-pxc/openshift.html>`__.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: nothing stored in Percona Software Repositories

File: source/ReleaseNotes/index.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.. _rel_notes:
+
+==================================================
+Percona Server for MongoDB |version| Release Notes
+==================================================
+
+.. toctree::
+   :maxdepth: 1
+
+   1.0.0
```
Comment (delgod) [inline source/ReleaseNotes/index.rst]: should we also add 0.3.0, 0.2.0 and 0.1.0 releases?

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Customizable my.cnf
+
+Simplified User Interface for Users to Interact with the Operator
+
+ * :jirabug:`CLOUD-182 <https://jira.percona.com/browse/CLOUD-182>`__ Automatic backup and restore provides a method of performing a hot backup of your MySQL data while the system is running. `Percona XtraBackup <https://www.percona.com/software/mysql-database/percona-xtrabackup>`__ is a free, online, open-source, backup tool.
+
+ * :jirabug:`CLOUD-212 <https://jira.percona.com/browse/CLOUD-212>`__ Uses the `xbcloud <https://www.percona.com/doc/percona-xtrabackup/2.3/xbcloud/xbcloud.html>`__ is part of Percona XtraDB Backup. The purpose of the xbcloud tool is to download and upload full or part of a `xbstream <https://www.percona.com/doc/percona-xtrabackup/2.3/
+
+ * :jirabug:`CLOUD-181 <https://jira.percona.com/browse/CLOUD-181>`__ Kubernetes expects communication within the cluster to be encrypted by default with TLS.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: Kubernetes doesn't expect anything.
please read carefully documentation https://www.percona.com/doc/kubernetes-operator-for-pxc/TLS.html
to understand what was implemented in CLOUD-181

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+ * :jirabug:`CLOUD-203 < https://jira.percona.com/browse/CLOUD-203>`__ Allows a `manual certificate signing request <https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/#create-a-certificate-signing-request-object-to-send-to-the-kubernetes-api>`__ to send to the Kubernetes API.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: completely wrong description, please read carefully documentation https://www.percona.com/doc/kubernetes-operator-for-pxc/TLS.html
to understand what was implemented in CLOUD-203

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+ * :jirabug:`CLOUD-193 <https://jira.percona.com/browse/CLOUD-193>`__ The ProxySQL service is configurable by changing available settings.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: please document this option https://jira.percona.com/browse/CLOUD-266
and provide a more detailed description here

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst (hunk 1 above)
Comment (patrickbirch) [reply source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: Would "The Operator provides a consistent Percona XtraDB Cluster instance." be more accurate?

File: source/ReleaseNotes/index.rst (hunk 3 above)
Comment (patrickbirch) [reply source/ReleaseNotes/index.rst]: Peter S said that we should start with 1.0.0

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst (hunk 1 above)
Comment (delgod) [reply source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: not an option
the next sentence also starts very similar "The Kubernetes Operators provide a consistent way to package..."

File: source/ReleaseNotes/index.rst [hunk 7]
```diff
@@ -0,0 +1,10 @@
+.. _rel_notes:
+
+==================================================
+Kubernetes Operator for XtraDB Cluster |version| Release Notes
```
Comment (delgod) [inline source/ReleaseNotes/index.rst]:
```suggestion
Kubernetes Operator for Percona XtraDB Cluster |version| Release Notes
```

OR

```suggestion
Percona Kubernetes Operator for Percona XtraDB Cluster |version| Release Notes
```

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 8]
```diff
@@ -0,0 +1,32 @@
+.. rn:: 1.0.0
+
+Percona Kubernetes Operator for XtraDB Cluster
+==============================================
+
+Percona announces the general availability of |Percona Kubernetes Operator for XtraDB Cluster| 1.0.0 on May 29, 2019. This release is now the current GA release in the 1.0 series. `Install the Kubernetes Operator for PXC by following the instructions <https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html>`__. Please see the `GA release announcement <https://www.percona.com/blog/2019/05/29/percona-kubernetes-operators/>`__. All of Percona's software is open-source and free.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: "Percona Kubernetes Operator for XtraDB Cluster" and "Kubernetes Operator for PXC"
both don't seem correct

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 9]
```diff
@@ -0,0 +1,32 @@
+.. rn:: 1.0.0
+
+Percona Kubernetes Operator for XtraDB Cluster
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]:
```suggestion
Percona Kubernetes Operator for Percona XtraDB Cluster
```

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+==============================================
+
+Percona announces the general availability of |Percona Kubernetes Operator for XtraDB Cluster| 1.0.0 on May 29, 2019. This release is now the current GA release in the 1.0 series. `Install the Kubernetes Operator for PXC by following the instructions <https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html>`__. Please see the `GA release announcement <https://www.percona.com/blog/2019/05/29/percona-kubernetes-operators/>`__. All of Percona's software is open-source and free.
+
+The Percona Kubernetes Operator for Percona XtraDB Cluster automates the lifecycle of your Percona XtraDB Cluster environment. The Operator can be used to instantiate a Percona XtraDB Cluster, or scale an existing Cluster.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.0.0.rst]: Percona XtraDB Cluster mentioned three times

**Pull request merged**